docs/product-features/staking/how-to.md

```diff
@@ -1,3 +1,80 @@
+
+<figure><img src="../../.gitbook/assets/staking/stake-my-staking.png" alt=""><figcaption></figcaption></figure>
+
+#### Staking Metrics
+
+* **Manual Staking:** The amount of $ALEX you are currently staking.&#x20;
+* **APR:** The Annual Percentage Rate of your investment. It is the interest you would earn by staking for one year
+* **Liquid Staking:** The amount of $ALEX you are currently staking through LISA.
+* **APY:** The Annual Percentage Yield of your investment. It is the interest you would earn by staking for one year, including compounding interest from re-staking your rewards.
+* **APower to be Distributed**: The amount of APower you will earn as a result of your investment.
+
+#### Cycles
+
+* **Cycles:** The number of the current cycle, as well as the numbers of the two upcoming cycles.
+* **Liquid Staking:** The amount of $ALEX you will be staking through LISA for each cycle as a result of your investments so far. 
+* **Manual Staking:** The amount of $ALEX you will be manually staking for each cycle as a result of your investments so far.
+
+By pressing "Proceed to LISA to Stake" you will open the dashboard for liquid staking on LISA. For more information, you can consult the [LISA Staking Guide](https://docs.lisalab.io/features-how-tos/staking-stacking).
+
+Once you've verified your current balances, APR, APY and cycle timing, you can proceed to the next step.
+
+### Step 3: Add ALEX Staking
+
+Select how much $ALEX you wish to stake and for how long. You can use the slider to personalize the amount of cycles, and you will see an estimate of how many days the selected amount of cycles will last. If you wish to stake ALEX tokens for different periods of time, you will need to execute two transactions. For example, if you wish to stake 100 ALEX for 8 cycles and 80 ALEX for 24 cycles, you will need to follow these steps twice. 
+
+<figure><img src="../../.gitbook/assets/staking/stake-config.png" alt=""><figcaption></figcaption></figure>
+
+The "MAX" button will stake your entire balance.
+
+In "Staking Details" you can see the numbers of the cycles you'll be staking for under your current settings, along with the corresponding "Start" and "End" blocks on the Stacks network. Afterwards, click "Stake" to begin staking.
+
+### Step 4: Confirm Staking
+
+You will be prompted to confirm that the selected settings are correct. Please verify the information on the pop-up window and press "Confirm" to continue.
+
+<figure><img src="../../.gitbook/assets/staking/stake-confirm.png" alt=""><figcaption></figcaption></figure>
+
+### Step 5: Confirm Transaction on your Wallet
+
+Your Stacks wallet will validate your identity and ask you to confirm the transaction. After that, wait for your transaction to be confirmed on the network.
```

Comment:
a que nos referimos con: "... will validate your identity..." ?

Reply:
Changed for "Your Stacks wallet will request your password and ask you to confirm the transaction."